packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,46 @@
+import { type TSESLint, type TSESTree } from '@typescript-eslint/utils';
+
+import { createRule, getParserServices } from '../util';
+
+type MessageIds = 'forbiddenFunctionSpread';
+
+export default createRule<[], MessageIds>({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description:
+        "Disallow spread operator that shouldn't be spread most of the time",
+      recommended: 'stylistic',
+      requiresTypeChecking: true,
+    },
+    messages: {
+      forbiddenFunctionSpread:
+        'Spreading a function is almost always a mistake. Did you forget to call the function?',
```

Comment:
[Proofreading] Maybe...

```suggestion
        'Spreading a function with no custom properties does nothing. Did you forget to call the function?',
```